pinot-controller/src/main/java/org/apache/pinot/controller/helix/core/SegmentDeletionManager.java

```diff
@@ -84,22 +86,28 @@ public void stop() {
     _executorService.shutdownNow();
   }
 
-  public void deleteSegments(final String tableName, final Collection<String> segmentIds) {
-    deleteSegmentsWithDelay(tableName, segmentIds, DEFAULT_DELETION_DELAY_SECONDS);
+  public void deleteSegments(String tableName, Collection<String> segmentIds) {
+    deleteSegments(tableName, segmentIds, _defaultDeletedSegmentsRetentionMillis);
   }
 
-  protected void deleteSegmentsWithDelay(final String tableName, final Collection<String> segmentIds,
-      final long deletionDelaySeconds) {
+  public void deleteSegments(String tableName, Collection<String> segmentIds,
+      long deletedSegmentsRetentionMillis) {
```

Comment:
(minor, same for other places)
```suggestion
      long deletedSegmentsRetentionMs) {
```